_posts/2019-06-25-off-by-one.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Berlin Hack & Tell \#72 - Off By One++
+date: 2019-06-25
+time: '18:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/Berlin-Hack-and-Tell/events/262505358/
+---
+
+* AFL
+* [webRTC beaker](https://github.com/D1plo1d/tegh-cli) by [Rob Gilson](https://github.com/D1plo1d)
+* [dotfile rage](https://gist.github.com/chr5tphr/506e4360f7569dcb1495cf63aaff9c90) by [Christopher](https://github.com/chr5tphr)
+* [Quadruped ROS2 Bot](https://hackaday.io/project/165423-quadruped-ros2-bot) by [Mario](https://github.com/qwqw330/)
+* [DebLibs](https://github.com/hellofresh/deblibs-gradle-plugin) by [Henry Addo](https://github.com/eyedol)
+* Crypto yield curve by Max
+* [Hexliterate](https://github.com/ssimono/hexliterate) by [Simon Alfassa](https://github.com/ssimono)  - **Hack of the month**
+* [https://github.com/KevinFrantz/infinito](infinito) by [https://github.com/KevinFrantz](Kevin Frantz)
+
```

Comment:
can you remove this line?

Reply:
Do you mean fix? I fixed it just now.

Reply:
no - I mean remove the blank line on the end

Reply:
Oh I see. Removed.